docs/how_to_guides/configuring_metadata_stores/how_to_configure_an_expectation_store_to_postgresql.rst

```diff
@@ -1,18 +1,127 @@
+        expectations_store_name: expectations_store
+
+        stores:
+            expectations_store:
+                class_name: ExpectationsStore
+                store_backend:
+                    class_name: TupleFilesystemStoreBackend
+                    base_directory: expectations/
+
+
+3. **Update your configuration file to include a new store for Expectations on PostgreSQL**
+
+    In our case, the name is set to ``expectations_postgres_store``, but it can be any name you like.  We also need to make some changes to the ``store_backend`` settings.  The ``class_name`` will be set to ``DatabaseStoreBackend``, and ``credentials`` will be set to ``${db_creds}``, which references the corresponding key in the ``config_variables.yml`` file.
+
+    .. code-block:: yaml
+
+        expectations_store_name: expectations_postgres_store
+
+        stores:
+            expectations_postgres_store:
+                class_name: ExpectationsStore
+                store_backend:
+                    class_name: DatabaseStoreBackend
+                    credentials: ${db_creds}
+
+
+4. **Confirm that the new Expectations store has been added by running** ``great_expectations store list``
+
+    Notice the output contains two Expectation stores: the original ``expectations_store`` on the local filesystem and the ``expectations_postgres_store`` we just configured.  This is ok, since Great Expectations will look for Expectations in PostgreSQL as long as we set the ``expectations_store_name`` variable to ``expectations_postgres_store``, which we did in the previous step.  The config for ``expectations_store`` can be removed if you would like.
+
+    .. code-block:: bash
+
+        great_expectations store list
+
+        - name: expectations_store
+        class_name: ExpectationsStore
+        store_backend:
+            class_name: TupleFilesystemStoreBackend
+            base_directory: expectations/
 
-If you want to be a real hero, we'd welcome a pull request. Please see :ref:`the Contributing tutorial <tutorials__contributing>` and :ref:`How to write a how to guide` to get started.
+        - name: expectations_postgres_store
+        class_name: ExpectationsStore
+        store_backend:
+            class_name: DatabaseStoreBackend
+            credentials:
+                database: '<your_db_name>'
+                drivername: postgresql
+                host: '<your_host_name>'
+                password: ******
+                port: '<your_port>'
+                username: '<your_username>'
+
+
+5. **Create a new Expectation Suite by running** ``great_expectations suite new``
+
+    This command prompts you to create and name a new Expectation Suite and to select a sample batch of data for the Suite to describe. Behind the scenes, Great Expectations will create a new table in your database called ``ge_expectations_store``, and populate the fields ``expectation_suite_name`` and ``value`` with information from the newly created Expectation Suite.
```

Comment:
Is this part clear? I feel like some information on what GE is doing behind the scenes is useful, but I dont want to add too much to basic how-to

Reply:
I think this makes sense to keep, and it is clear. It is useful to the user to know where in the database the expectations are written. I'm assuming the table/column names are the default ones, and that the table name and column names may or may not be configurable at this time.